docs/notebooks/dispersion/stix_dispersion.ipynb

```diff
@@ -0,0 +1,768 @@
+   "execution_count": null,
+   "id": "7902e8ff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt_tf_hot = {}\n",
+    "for key, val in w_tf_hot.items():\n",
+    "    plt_tf_hot[key] = norm_bellan_plot(**{**hot_inputs, \"w\": val})\n",
+    "\n",
+    "plt_tf_cold = {}\n",
+    "for key, val in w_tf_cold.items():\n",
+    "    plt_tf_cold[key] = norm_bellan_plot(**{**cold_inputs, \"w\": val})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93bab73c",
+   "metadata": {},
+   "source": [
+    "Stix needs to recalculated using the Bellan inputs as a base."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9fca167a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stix_inputs = {**base_inputs, \"ions\": [base_inputs[\"ion\"]]}\n",
+    "del stix_inputs[\"k\"]\n",
+    "del stix_inputs[\"ion\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57469172",
+   "metadata": {},
+   "source": [
+    "Bellan fixes $k$ and then calculates $\\omega$ for each mode and propagation angle $\\theta$. This means we cannot simply take solutions from Bellan and get corresponding $k$ values via Stix. In order to solve this problem we need to create a version of `stix()` that can be optimized by `scipy.optimize.root_scalar()`.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "724eea0f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# partially bind plasma parameter keywords to stix()\n",
+    "_opt = stix_inputs.copy()\n",
+    "del _opt[\"theta\"]\n",
+    "stix_partial = functools.partial(stix, **_opt)\n",
+    "\n",
+    "\n",
+    "def stix_optimize(w, theta, mode, k_expected):\n",
+    "    w = np.abs(w)\n",
+    "    results = stix_partial(w=w * u.rad / u.s, theta=theta * u.rad).value\n",
+    "\n",
+    "    # only consider real and positive solutions\n",
+    "    real_mask = np.where(np.imag(results) == 0, True, False)\n",
+    "    pos_mask = np.where(np.real(results) > 0, True, False)\n",
+    "    mask = np.logical_and(real_mask, pos_mask)\n",
+    "\n",
+    "    # get the correct k to compare\n",
+    "    if np.count_nonzero(mask) == 1:\n",
+    "        results = np.real(results[mask][0])\n",
+    "    elif mode == \"fast_mode\":\n",
+    "        # fast_mode has a larger phase velocity than\n",
+    "        # the alfven_mode, thus take the smaller k-value\n",
+    "        results = np.min(np.real(results[mask]))\n",
+    "    else:  # alfven_mode\n",
+    "        results = np.max(np.real(results[mask]))\n",
+    "\n",
+    "    return results - k_expected"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb4e6ace",
+   "metadata": {},
+   "source": [
+    "Let's use the Cold case Bellan solution to solve for the Stix solution.  Note only the `fast_mode` and `slow_mode` solutions are being used to seed the Stix solution, because the `acoustic_mode` disappears in the cold plasma limit."
```

Comment:
```suggestion
    "Let's use the Cold case Bellan solution to solve for the Stix solution.  Note only the `fast_mode` and `slow_mode` solutions are being used to seed the Stix solution because the `acoustic_mode` disappears in the cold plasma limit."
```